doc/man3/SSL_get0_peer_rpk.pod

```diff
@@ -58,6 +58,7 @@ The SSL_add_expected_rpk() function is a wrapper around
 L<SSL_dane_tlsa_add(3)>.
 When DANE is enabled, the configured TLSA records will be used to validate the
 peer's public key or certificate.
+If DANE is not enabled, then no validation will occur.
```

Comment:
And the validation will default to IIRC untrusted, right?  The application verify callback will have to also have set SSL_VERIFY_NONE or a callback that always returns ok=1 to not then fail...

Reply:
Correct.

Reply:
Perhaps worth a sentence to that effect. Otherwise, a user might conclude that with DANE not enabled the connection will just succeed unvalidated by default (of course SSL_VERIFY_NONE is still a default IIRC, but that is not the case in most user applications).